sdk/core/azure_core/src/credentials.rs

```diff
@@ -65,6 +106,17 @@ impl Debug for Secret {
     }
 }
 
+impl Debug for CosmosConnectionString {
```

Comment:
Just derive `SafeDebug` on the type. You also shouldn't format for `Debug`. That's more for `Display`, which isn't needed here.